build.gradle

```diff
@@ -58,6 +60,38 @@ allprojects {
     repositories {
         jcenter()
         google()
+        maven {
+            url "http://packages.geotracinternational.com/nexus/repository/maven-snapshots/"
+            credentials {
+                username = "${nexus_username}"
+                password = "${nexus_password}"
+            }
+        }
+        maven {
+            url "http://packages.geotracinternational.com/nexus/repository/maven-releases/"
+            credentials {
+                username = "${nexus_username}"
+                password = "${nexus_password}"
+            }
+        }
+    }
+
+    // Add this to reduce caching to avoid stale snapshots
+    configurations.all {
+        resolutionStrategy.cacheChangingModulesFor 60, 'seconds'
+        resolutionStrategy.cacheDynamicVersionsFor 1, 'hours'
+
+        resolutionStrategy.eachDependency { DependencyResolveDetails details ->
```

Comment:
Not sure if we want this block for this project as it doesn't seem like we are manipulating the version number.